docs/manuals/core/entities/scenario-cycle-mgt.md

```diff
@@ -76,6 +76,25 @@ scenario == scenario_retrieved
 
 Here, the two variables `scenario` and `scenario_retrieved` are equal.
 
+# Get scenarios by config id
+
+Scenarios can also be retrieved by their config_id by calling `taipy.get_entities_by_config_id()^` with their config_id.
```

Comment:
"by their config_id by calling \`taipy.get_entities_by_config_id()^\` with their config_id" => "using the method  \`taipy.get_entities_by_config_id()^\`, providing the configuration identifier"